tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
```

Comment:
Sztem lehet false a strict mode. Azt gyanitom emiatt hal el a rendereles mar akkor is ha csak ts figyelmeztetesed van, de nincs hiba. Nem vagyok tul jo tsconfigban, de ha Pilu is nagyjabol egyetert, akkor allitsd es szebb lesz onnantol az eleted.